### PR TITLE
Fix for issue #1224 showing complete notification on click 

### LINKED ISSUE #1224: Show complete text of welcome notification
**Summary:**
Welcome Notification doesn't redirect the user to any link. As a user, I would like to know the complete information displayed in the notification. So, Either it should redirect the user to an external link containing the full information or it should show the complete text of the notification.

**Steps to reproduce:** 
1. After Login, click on Notifications.
2. Check the welcome notification.

**Device and Android version:** 
Samsung A8
Android 6.0 Marshmallow
 
 **Commons app version:** 
2.6.7

**Screen-shots:** 

**How will I proceed on this issue**
Since the API itself doesn't provide any link for welcome notifications. Removing the maxLines limit of title can be a quick fix. @maskaravivek please let me know if this approach is okay, will open the PR accordingly.
Thanks. :)

### PR DESCRIPTION
**Pull request by Jatin0312**
## Description

Fixes #1224
Created custom Text View for Title and description which will be clickable and displays complete text if clicked.

## Screenshots showing what changed
 
solved3

### CODE REVIEW

**Comment by Jatin0312** [comment]
Please review and provide some suggestion.
Thanks.

**Comment by misaochan** [comment]
Looks great to me! :) Should probably add a whitespace between the text and "show less" though.

**Comment by Jatin0312** [comment]
Sure I will make the required changes.

**Comment by Jatin0312** [comment]
Added Whitespace between show less and text.
**Screenshot**

solved6

File: gradle.properties [hunk 1]
```diff
@@ -1,21 +1,29 @@
-gradleVersion = 3.0.0
```
Comment (maskaravivek) [inline gradle.properties]: Why have so many things changes in the `gradle.properties`

**Comment by Jatin0312** [comment]
Actually most of the added things is commented and android studio kind of reformated it.
I will make sure from next time this doesn't happen next time.

**Pull request merged**